README.md

```diff
@@ -1,6 +1,6 @@
-# web3-react-magic-connect
+# web3-react-magic-universal-wallet
 
-[Magic Connect](https://magic.link/docs/connect/overview) connector for [web3-react](https://github.com/Uniswap/web3-react).
+[Magic Universal Wallet](https://magic.link/docs/universal/overview) connector for [web3-react](https://github.com/Uniswap/web3-react).
```

Comment:
The Universal tab has been removed. We should probably point this to https://magic.link/docs/wallets/wallet-types#universal-wallet instead.